float64 z
bool slow
---
bool success
string message
